lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -554,19 +691,25 @@ def _old_flow_stats(self, stats_msgs):
                 if rule_id == "":
                     continue
 
-                current_ver = \
-                    self._session_rule_version_mapper.get_version(sid,
-                                                                  ipv4_addr,
-                                                                  rule_id)
+                ng_session_id = _get_ng_enable(stat)
+                if ng_session_id and ng_session_id != REG_ZERO_VAL:
+                    current_ver = \
```

Comment:
ng_session_id updates with every rule update correct? So we don't need the normal rule_version?

Reply:
If ng_session_id==rule_version for 5g I wonder if we can just reuse the version register? With the recent changes sessiond passes version to pipelined so its no longer managed on pipelined. I wonder if we can nicely combine things because of this. cc @themarwhal